### PR TITLE
feat(ldlc): adding ldlc french store

### PR DESCRIPTION
**Pull request by MrToshio**

### CODE REVIEW

**Review by jef — COMMENTED** [review]
This is great! Do you mind adding to the documentation?

Thanks!

**Comment by MrToshio** [comment]
> This is great! Do you mind adding to the documentation?
> 
> Thanks!

Oops, done! :-)

File: src/store/model/ldlc.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
+import {Store} from './store';
+
+export const Ldlc: Store = {
+  currency: '€',
+  labels: {
+    inStock: {
+      container: '.stock-1',
+      text: ['en', 'stock'],
```
Comment (jef) [inline src/store/model/ldlc.ts:8]:
```suggestion
      container: '.stock',
      text: ['stock'],
```

Perhaps we just take this change?

Comment (jef) [inline src/store/model/ldlc.ts:8]:
```suggestion
      container: '.stock',
      text: ['stock'],
```

Perhaps we just take this change?

File: src/store/model/ldlc.ts [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    },
+    maxPrice: {
+      container: '.price .price',
+      euroFormat: true,
+    },
+    outOfStock: {
+      container: '.stock-4 .stock-7',
+      text: ['RUPTURE'],
+    },
```
Comment (jef) [inline src/store/model/ldlc.ts:17]:
```suggestion
    outOfStock: {
      container: '.stock',
      text: ['rupture'],
    },
```

File: src/store/model/ldlc.ts [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+  },
+  links: [
+    {
+      brand: 'evga',
+      model: 'xc3 ultra',
+      series: '3080',
+      url: 'https://www.ldlc.com/fr-be/fiche/PB00369303.html',
+    },
```
Comment (jef) [inline src/store/model/ldlc.ts:25]:
```suggestion
    {
      brand: 'test:brand',
      model: 'test:model',
      series': 'test:series',
      url: 'https://www.ldlc.com/fr-be/fiche/PB00317660.html',
    },
    {
      brand: 'evga',
      model: 'xc3 ultra',
      series: '3080',
      url: 'https://www.ldlc.com/fr-be/fiche/PB00369303.html',
    },
```

We should add a test model in here.

**Review by jef — APPROVED** [review]
Great! Let's apply these changes and merge it in!

File: src/store/model/ldlc.ts (hunk 1 above)
Comment (MrToshio) [reply src/store/model/ldlc.ts]: updated, I think it's ready to merge

**Review by jef — APPROVED** [review]
Great! Thank you very much!

**Comment by MrToshio** [comment]
> Great! Thank you very much!

nonono, thank YOU very much! :D

**Comment by tirrorex** [comment]
Hello @MrToshio ,
can you confirm that the bot is only checking for 3080's? 
If so i can make another pull request based on your code for 3070 and 3060ti

**Comment by jef** [comment]
Feel free to make a PR. Looks like it's only 3080s.

**Comment by MrToshio** [comment]
> Hello @MrToshio ,
> can you confirm that the bot is only checking for 3080's?
> If so i can make another pull request based on your code for 3070 and 3060ti

Hello, I confirm that I only added the 3080, but yeah feel free to add any other one you feel like adding, don't hesitate to contact me if you are other questions 👍

**Pull request merged**